README.md

```diff
@@ -26,52 +25,20 @@ Since anything in our codebase can be extended, overwritten, or installed as a p
 
 # Getting started
 
-### Requirements
-
-Reaction requires Meteor, Git, Mongo DB, OS-specific build tools. For step-by-step instructions, follow the documentation for [OS X](https://docs.reactioncommerce.com/docs/next/installation-osx), [Windows](https://docs.reactioncommerce.com/docs/next/installation-windows) or [Linux](https://docs.reactioncommerce.com/docs/next/installation-linux).
-
-### Install and run with CLI
-
-Install the [Reaction CLI](https://github.com/reactioncommerce/reaction-cli) to get started with Reaction:
-
-```sh
-npm install -g reaction-cli
-```
-
-Create your store:
-
-```sh
-reaction init
-cd reaction
-reaction
-```
-
-Open `localhost:3000`
-
-Learn more on how to [configure your project](https://docs.reactioncommerce.com/reaction-docs/master/configuration).
-
-Having installation issues? Check out our [troubleshooting docs](https://docs.reactioncommerce.com/docs/next/troubleshooting-development).
-
-### Install and run with Docker
-
-You can also run the app locally using [`docker-compose`](https://docs.docker.com/compose/) by running:
-
-```sh
-docker network create api.reaction.localhost
-docker-compose up
-```
+## Reaction 2.0
```

Comment:
Thinking now that we should eliminate the `2.0 and `pre-2.0` distinctions and just direct to the platform. We can also say something like "Looking for installation docs for an older version? https://docs.reactioncommerce.com/versions" or similar?